index.bs

```diff
@@ -10056,35 +10174,60 @@ interface AudioWorkletProcessor {
 
 <h5 id="AudioWorketProcessor-constructors">
 Constructors</h5>
+
 <dl dfn-type="constructor" dfn-for="AudioWorkletProcessor">
 	: <dfn>AudioWorkletProcessor(options)</dfn>
 	::
-		When the constructor for {{AudioWorkletProcessor}} is invoked, the following steps are performed:
+		When the constructor for {{AudioWorkletProcessor}} is invoked,
+		the following steps are performed on the <a>rendering thread</a>.
+
+		If any of these steps throws an exception, abort the rest of
```

Comment:
I think this paragraph should probably be placed inside the algorithm block, perhaps after the first paragraph of the algorithm.

Reply:
Done.